### DISCUSSION TITLE
"Is there an estimated date for the stable version 3 of Riverpod?

### DISCUSSION THREAD

**Discussion by MiniSuperDev** [rrousselGit/riverpod#3512] [Q&A]
In my team we want to use some features from version 3, such as generics, [mixins on notifiers](https://github.com/rrousselGit/riverpod/issues/2576), sealed async value.

But we don't want to migrate until it is stable, do you have an estimated date?

Or do you recommend going to production with version 3 dev.

Thanks

**Comment by rrousselGit** [comment]
Somewhere during the second part of the year. Or maybe once macros are released.
